src/rebar_prv_shell.erl

```diff
@@ -331,8 +331,8 @@ reread_config(State) ->
     case find_config(State) of
         no_config ->
             ok;
-        ConfigList ->
-            _ = rebar_utils:reread_config(ConfigList),
+        Config ->
+            _ = rebar_utils:reread_config([Config]),
```

Comment:
Since that's a public (not `_prv_`) API, the single-item-list compatibility is probably better placed at `consult_config`, really, but I'd like to hear your word.